Templates/HttpPUT(CRUD)-CSharp/run.csx

```diff
@@ -0,0 +1,31 @@
+﻿#r "Microsoft.WindowsAzure.Storage"
+
+using System.Net;
+using Microsoft.WindowsAzure.Storage.Table;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, CloudTable outTable, TraceWriter log)
+{
+    dynamic data = await req.Content.ReadAsAsync<object>();
+    string name = data?.name;
+
+    if (name == null)
+    {
+        return req.CreateResponse(HttpStatusCode.BadRequest, "Please pass a name in the request body");
+    }        
+
+    Person updatedPerson = new Person()
```

Comment:
I think it might be more natural for this method to take a person as the input payload, and do the insert or replace. I.e., GET to retreive a person, update one or more properties client side, then call this PUT function to apply the updates.